File: EvenNumbers/cpp/EvenNumbers.cpp
// Even Numbers solution in C++ for CodeEval.com by Steven A. Dunn

#include <cstdlib>
#include <fstream>
#include <iostream>

using namespace std;

int main(int argc, char* argv[])
{
	ifstream inputFile(argv[1]);
	string line;
	
	if (inputFile)
	{
		while (getline(inputFile, line))
		{
			int n = atoi(line.c_str());
			if (n % 2 == 0) { cout << "1" << endl; }
			else { cout << "0" << endl; } 
		}
	}
}

File: EvenNumbers/c#/EvenNumbers.cs
// even numbers soln in c# for code eval by steven a dunn

using System;
using System.IO;

class FindAWriter {
  public static void Main(string[] args) {
    using (StreamReader reader = new StreamReader(args[0])) {
      while (!reader.EndOfStream) {
        string line = reader.ReadLine();
        if (null == line)
          continue;

        int n = Int32.Parse(line);
        if (n % 2 == 0)
          Console.WriteLine(1);
        else
          Console.WriteLine(0);
      }
    }
  }
}